src/System.IO.Compression/src/System/IO/Compression/ZipArchiveEntry.cs

```diff
@@ -39,6 +39,7 @@ public partial class ZipArchiveEntry
         private bool _currentlyOpenForWrite;
         private bool _everOpenedForWrite;
         private Stream _outstandingWriteStream;
+        private uint _externalFileAttr;
```

Comment:
Is it still correct to use `uint` internally? Does it matter? 
It's a bit odd to use it and then test with int.MaxValue.